.travis.yml

```diff
@@ -2,6 +2,12 @@ language: ruby
 before_install:
   - gem update bundler
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.2.2
+  - 2.2.5
```

Comment:
Also, why is ruby 2.1 not tested anymore? According to their site it's still in security maintenance: https://www.ruby-lang.org/en/downloads/